core/src/main/scala/fs2/time/time.scala

```diff
@@ -30,7 +31,7 @@ package object time {
    * @param S           Strategy to run the stream
    * @param scheduler   Scheduler used to schedule tasks
    */
-  def awakeEvery[F[_]](d: FiniteDuration)(implicit F: Async[F], FR: Async.Run[F], S: Strategy, scheduler: Scheduler): Stream[F,FiniteDuration] = {
+  def awakeEvery[F[_]](d: FiniteDuration)(implicit F: Async[F], S: Strategy, scheduler: Scheduler): Stream[F,FiniteDuration] = {
```

Comment:
OK sure -- I'll remove the Strategy param from Scheduler and use `start` here instead. I'll also add calls to `start` in each of the uses of completion callbacks in io package.